src/domain/transaction.ts

```diff
@@ -1,57 +1,30 @@
-import type { NetworkString, UnblindedOutput } from 'ldk';
-
-export type UtxosAndTxsByNetwork = Record<NetworkString, UtxosAndTxs>;
-
-export interface UtxosAndTxs {
-  // outpoint string -> UnblindedOutput
-  utxosMap: Record<string, UnblindedOutput>;
-  transactions: TxsHistory;
-}
-
-export type TxsHistory = Record<TxDisplayInterface['txId'], TxDisplayInterface>;
+export type UnblindingData = {
```

Comment:
You could use `UnblindOutputResult` type from LiquidJS

Reply:
right. I made this on purpose, I wanted to have a JSON-friendly type in order to drop our custom (de)serializers made for the `Buffer` type.